interfaces/cython/cantera/test/test_reactor.py

```diff
@@ -1516,7 +1516,7 @@ class CombustorTestImplementation:
     """
 
     def setUp(self):
-        self.referenceFile = pjoin(os.path.dirname(__file__), 'data', 'CombustorTest-integrateWithAdvance.csv')
+        self.referenceFile = Path(__file__).parent / "data" / "CombustorTest-integrateWithAdvance.csv"
```

Comment:
IMO in cases where more than one path is being added, `joinpath()` is clearer.

Reply:
But isn't this the `test_data_dir`?

Reply:
Yup, but it doesn't inherit from `utilities` ...

Reply:
I added a module constant `utilities.TEST_DATA_PATH` that is also assigned to `utilities.CanteraTest.test_data_path`. I am using the latter throughout, as it's consistent with the original `utilities.CanteraTest.test_data_dir`.